modules/phase_field/src/action/EulerAngle2RGBAction.C

```diff
@@ -25,6 +25,7 @@ validParams<EulerAngle2RGBAction>()
   params.addParam<std::string>("auxvariable_name_base", "RGB", "Base name of the auxvariables");
   params.addClassDescription("Set up auxvariables and auxkernels to output Euler angles as RGB "
                              "values interpolated across inverse pole figure");
+  params.addParam<unsigned int>("phase", 0, "The phase to use for all queries. (Default: 0 ALL)");
```

Comment:
make this an optional parameter and use isParamValid or isParamSetByUser instead